docs/advanced/classes.rst

```diff
@@ -562,7 +562,7 @@ crucial that instances are deallocated on the C++ side to avoid memory leaks.
 
 Destructors that call Python
 ============================
-
+earch);45M
```

Comment:
That doesn't look right :thinking:

Reply:
```suggestion

```

Reply:
:#O Oops! Will restore the lost space tomorrow. Wasn't even something I was editing...